viewer/apiCrons.js

```diff
@@ -166,6 +166,15 @@ class CronAPIs {
             }
           }
 
+          // Convert notifier to array for consistency (backward compatibility for string values)
+          if (result.notifier) {
```

Comment:
@awick i want to support string values as well as arrays, but i'm not sure if this is the best way to do it